### PR TITLE
Pathch 6 main page ui change

### PR DESCRIPTION
**Pull request by SARTHAK1SINGH**
Help section updated according to this page. ON window load show added
Background changed.
Typing effect added to project title.
shadow effect added to form box on mic button click.

### CODE REVIEW

**Comment by SARTHAK1SINGH** [comment]
Annotation 2020-08-20 214834

#on mic click

Annotation 2020-08-20 214907

File: templates/index.html [hunk 1]
```diff
@@ -284,8 +307,9 @@ <h5 class="modal-title" id="exampleModalLongTitle" style="color:black">Understan
 
         <div class="header">
             <form autocomplete="off"  method = "POST" action = "/">
-                <h1 class="title"><i class="fa fa-hospital" aria-hidden="true"></i> The Cure Origin <i class="fa fa-heartbeat" aria-hidden="true"></i></h1>
-                <div class="formbox">
+
+                <h1 class="title"><i class="fa fa-hospital" aria-hidden="true"></i> The Cure-Mate <i class="fa fa-heartbeat" aria-hidden="true"></i></h1>
```
Comment (maze-runnar) [inline templates/index.html:311]:
```suggestion
                <h1 class="title"> Cure-Mate <i class="fa fa-heartbeat" aria-hidden="true"></i></h1>
```

File: templates/index.html [hunk 2]
```diff
@@ -263,7 +286,7 @@ <h5 class="modal-title" id="exampleModalLongTitle" style="color:black">Understan
        <a class="nav-link" href="{{url_for('history')}}">History</a>
       </li>
       <li class="nav-item">
-        <button type="button" class="btn btn-outline-info" data-toggle="modal" data-target="#exampleModalCenter" style="border:none">
+        <button type="button" id="Help" class="btn btn-outline-info" data-toggle="modal" data-target="#exampleModalCenter" style="border:none">
```
Comment (maze-runnar) [inline templates/index.html:289]:
```suggestion
        <button type="button" id="Help" class="btn btn-outline-dark" data-toggle="modal" data-target="#exampleModalCenter" style="border:none;color:white">
```

File: templates/index.html [hunk 3]
```diff
@@ -284,8 +307,9 @@ <h5 class="modal-title" id="exampleModalLongTitle" style="color:black">Understan
 
         <div class="header">
             <form autocomplete="off"  method = "POST" action = "/">
-                <h1 class="title"><i class="fa fa-hospital" aria-hidden="true"></i> The Cure Origin <i class="fa fa-heartbeat" aria-hidden="true"></i></h1>
-                <div class="formbox">
+
+                <h1 class="title"><i class="fa fa-hospital" aria-hidden="true"></i> The Cure-Mate <i class="fa fa-heartbeat" aria-hidden="true"></i></h1>
+                <div class="formbox" id="formbox">
                     <div class="form-group autocomplete">
                     <label for="exampleInputEmail1">Disease</label>
```
Comment (maze-runnar) [inline templates/index.html:314]:
```suggestion
                   <label for="exampleInputEmail1">Disease(<small><i class="fa fa-microphone"></i> ex. my disease is Heart Attack</small>)</label>
```

**Pull request merged**